README.md

```diff
@@ -159,7 +159,6 @@ if(CATKIN_ENABLE_TESTING AND ENABLE_COVERAGE_TESTING)
   include(CodeCoverage)
   APPEND_COVERAGE_COMPILER_FLAGS()
   set(COVERAGE_EXCLUDES "*/test/*")
```

Comment:
I'm pretty sure that the `COVERAGE_EXCLUDES` variable has no effect here. This is only relevant for use with the coverage report target created by `add_code_coverage()`. But this we don't use here, because we use the bash script provided by codecov.io.
Did you validated the effect of `COVERAGE_EXCLUDES`?

Reply:
You are right :(.  If you look at the files that were "tested" in the above linked codecov.io report you will find tests.  Good catch.  I think we'll need to bring back the codecov yaml file.  I'll test that now and update this.